tests/business/notebookrunner_test.py

```diff
@@ -76,7 +76,7 @@ async def test_run(
             "business": {
                 "failure_count": 0,
                 "name": "NotebookRunner",
-                "notebook": "test-notebook.ipynb",
+                "notebook": ANY,
```

Comment:
I think we've now lost this test, in that nothing is now testing that the `notebook` data is set correctly to the executing notebook. It's some duplication, but it might be a bit cleaner to add a separate test for the recursive notebook running that uses a separate directory.